.artifactignore

```diff
@@ -1,7 +0,0 @@
-*.js.map
```

Comment:
what's the reason to remove this file ?

Reply:
@gunzip That file is (only) used in a Azure DevOps task for publishing the artefacts in Azure DevOps (I added it the previous pipeline)... In the new pipeline I removed this task and filtered the files to include/exclude in another way. So, it is no more used.